charts/producer-app/values.yaml

```diff
@@ -1,5 +1,7 @@
 nameOverride: bakdata-producer-app
 
+replicaCount: 1
+
```

Comment:
It makes sense to set a default of 1, but please remove the comment in lines 11-12 then

Reply:
Does it make sense? It only applies if deployment is enabled and that isn't the default. 

If you keep it, it should be updated in the docs

Reply:
As this is unrelated to the actual change in this PR, maybe just leave it as it was